timeline/graph.go

```diff
@@ -774,7 +774,7 @@ type ItemRow struct {
 }
 
 func (ir ItemRow) hasContent() bool {
-	return ir.DataID != nil || ir.DataText != nil || ir.DataFile != nil || !ir.Location.IsEmpty()
+	return ir.DataID != nil || ir.DataText != nil || ir.DataFile != nil || !ir.IsEmpty()
```

Comment:
I prefer `ir.Location.` since it becomes clear that we are checking if the _Location_ is empty.